core/trino-main/src/test/java/io/trino/operator/aggregation/TestTypedHistogram.java

```diff
@@ -59,4 +60,85 @@ public void testMassive()
             assertEquals(BIGINT.getLong(outputBlock, i + 1), BIGINT.getLong(outputBlock, i));
         }
     }
+
+    @Test
+    public void testGetPositionCountBasic()
+    {
+        Block inputBlock = inputBlockFromNumbers(Arrays.asList(1, 2, 3));
+
+        SingleTypedHistogram typedHistogram = new SingleTypedHistogram(
+                BIGINT,
+                BLOCK_TYPE_OPERATORS.getEqualOperator(BIGINT),
+                BLOCK_TYPE_OPERATORS.getHashCodeOperator(BIGINT),
+                inputBlock.getPositionCount());
+
+        addInputBlockToTypedHistogram(typedHistogram, inputBlock);
+        assertEquals(typedHistogram.getPositionCount(), 3);
+    }
+
+    @Test
+    public void testGetPositionCountDuplicates()
+    {
+        Block inputBlock = inputBlockFromNumbers(Arrays.asList(1, 2, 1));
+
+        SingleTypedHistogram typedHistogram = new SingleTypedHistogram(
+                BIGINT,
+                BLOCK_TYPE_OPERATORS.getEqualOperator(BIGINT),
+                BLOCK_TYPE_OPERATORS.getHashCodeOperator(BIGINT),
+                inputBlock.getPositionCount());
+
+        addInputBlockToTypedHistogram(typedHistogram, inputBlock);
+        assertEquals(typedHistogram.getPositionCount(), 2);
+    }
+
+    @Test
+    public void testGetPositionCountLargeExpected()
+    {
+        Block inputBlock = inputBlockFromNumbers(Arrays.asList(1, 2, 3), 99);
+
+        SingleTypedHistogram typedHistogram = new SingleTypedHistogram(
+                BIGINT,
+                BLOCK_TYPE_OPERATORS.getEqualOperator(BIGINT),
+                BLOCK_TYPE_OPERATORS.getHashCodeOperator(BIGINT),
+                99);
+
+        addInputBlockToTypedHistogram(typedHistogram, inputBlock);
+        assertEquals(typedHistogram.getPositionCount(), 3);
+    }
+
+    @Test
+    public void testGetPositionCountEmpty()
+    {
+        Block inputBlock = inputBlockFromNumbers(Arrays.asList());
+
+        SingleTypedHistogram typedHistogram = new SingleTypedHistogram(
+                BIGINT,
+                BLOCK_TYPE_OPERATORS.getEqualOperator(BIGINT),
+                BLOCK_TYPE_OPERATORS.getHashCodeOperator(BIGINT),
+                1);
+
+        addInputBlockToTypedHistogram(typedHistogram, inputBlock);
+        assertEquals(typedHistogram.getPositionCount(), 0);
+    }
+
+    private Block inputBlockFromNumbers(List<Integer> numbersToInsert)
+    {
+        return inputBlockFromNumbers(numbersToInsert, numbersToInsert.size());
+    }
+
+    private Block inputBlockFromNumbers(List<Integer> numbersToInsert, int expectedEntries)
+    {
+        BlockBuilder inputBlockBuilder = BIGINT.createBlockBuilder(null, expectedEntries);
+        for (int n : numbersToInsert) {
+            BIGINT.writeLong(inputBlockBuilder, n);
+        }
+        return inputBlockBuilder.build();
+    }
```

Comment:
You can use `BlockAssertions#createLongsBlock`

Reply:
Thanks for the suggestion, I removed the `inputBlockFromNumbers` helper functions and replaced them with `BlockAssertions#createLongsBlock`.